contrib/examples/prometheus/README.md

```diff
@@ -0,0 +1,87 @@
+# Prometheus Metrics
+
+Service Catalog maintains metrics through the Prometheus client library and
+exposes them through the Prometheus http adapter at the Controller's /metrics
+endpoint. These metrics can be accessed directly via HTTP GET or more commonly
+scraped by the Prometheus monitoring application which persists metrics and
+facilitates analysis through a Web UI and powerful query language.
+
+Many metrics are not created and exposed until Service Catalog performs
+operations which would impact the metrics.  IE if you have no Service Brokers
+defined, there will be no metrics around class or plan count and likely no OSB
+Client operation metrics.  So before proceeding, it's recommended you have a
+Broker defined and created a Service Instance.
+
+To view the raw metrics:
+
+```
+# setup a port forward so we can curl against Controller Manager
+
+$ kubectl get pods -l app=catalog-catalog-controller-manager -n catalog -o name | \
+    sed 's/^.*\///' | xargs -I{} kubectl port-forward {} -n catalog 8089:8080 &
+
+$ curl -s http://localhost:8089/metrics  | grep servicecatalog
```

Comment:
this drops us into a pseudo shell of some sort?

Reply:
The first command actually backgrounds a port forward allowing you to hit http://localhost:8089 and kube will forward the request to the catalog-controller pod's end point.